bot/cogs/moderation.py

```diff
@@ -1088,8 +1089,9 @@ def _infraction_to_string(self, infraction_object):
         )
 
         icon_url = INFRACTION_ICONS.get(infr_type, Icons.token_removed)
-        embed.set_author(name="Infraction Information", icon_url=icon_url)
-        embed.set_footer(text=f"Please review our rules over at https://pythondiscord.com/about/rules")
+        embed.set_author(name="Infraction Information", icon_url=icon_url, url=RULES_URL)
+        embed.title = f"Please review our rules over at {RULES_URL}"
```

Comment:
I don't know if I like `over at` I prefer `at`
```suggestion
        embed.title = f"Please review our rules at {RULES_URL}"
```

Reply:
I just noticed this is what was there before actually, disregard this.